cloudfoundry/provider.go

```diff
@@ -87,6 +87,7 @@ func Provider() terraform.ResourceProvider {
 			"cloudfoundry_isolation_segment": dataSourceIsolationSegment(),
 			"cloudfoundry_space":             dataSourceSpace(),
 			"cloudfoundry_service_instance":  dataSourceServiceInstance(),
+			"cloudfoundry_user_provided_service_instance":  dataSourceUserProvidedServiceInstance(),
```

Comment:
data source name should be `cloudfoundry_user_provided_service` and it is not `go fmt`

Reply:
Why not with _instance suffix?
The CF client library also uses the method name GetUserProvidedServiceInstances.

Reply:
resource is named `cloudfoundry_user_provided_service` that's only to be equivalent